portal/views/portal.py

```diff
@@ -919,6 +919,13 @@ def spec():
             },
         },
         "schemes": ("http", "https"),
+        "securityDefinitions": {
+            "Authorization": {
```

Comment:
Can you rename this to `ServiceToken`? `Authorization` can be an overloaded term